source/index.html

```diff
@@ -198,19 +212,19 @@ <h2 class="visually-hidden">Наши контакты.</h2>
           Свяжитесь с нами!
         </p>
         <dl class="contacts__list">
-          <div class="contacts__item">
+          <div class="contacts__item contacts__item--phone">
             <dt class="contacts__item-title">Звоните:</dt>
             <dd class="contacts__info">
               <a class="contacts__link" href="tel:+78125556666">+7 (812) 555-66-66</a>
             </dd>
           </div>
-          <div class="contacts__item">
+          <div class="contacts__item contacts__item--mail">
             <dt class="contacts__item-title">Пишите:</dt>
             <dd class="contacts__info">
               <a class="contacts__link" href="mailto:mail@htmlacademy.ru">mail@htmlacademy.ru</a>
             </dd>
           </div>
-          <div class="contacts__item">
+          <div class="contacts__item contacts__item--adress">
```

Comment:
Ох уж это странное слово английское...
```suggestion
          <div class="contacts__item contacts__item--address">
```